generic/tests/iozone_windows.py

```diff
@@ -50,10 +51,46 @@ def get_driver():
                 driver_name = None
         return driver_name
 
+    def run_iozone_parallel(timeout):
+        """ Run the iozone parallel. """
+        iozone_sessions = []
+        iozone_threads = []
+        thread_maps = {}
+
+        for _ in range(len(disk_letters)):
```

Comment:
`for _ in disk_letters:` as disk_letters is a list

Reply:
yes, you are right, I forgot it. thanks.